crates/compilers/src/compilers/resolc/compiler.rs

```diff
@@ -126,84 +126,99 @@ impl Resolc {
         }
     }
 
-    pub fn install(resolc_version: Option<&Version>, solc_compiler: SolcCompiler) -> Result<Self> {
-        let solc_version = match &solc_compiler {
+    pub fn install(_resolc_version: Option<&Version>, solc_compiler: SolcCompiler) -> Result<Self> {
+        let _solc_version = match &solc_compiler {
             SolcCompiler::Specific(solc) => Some(solc.version_short()),
             #[cfg(feature = "svm-solc")]
             SolcCompiler::AutoDetect => None,
         };
-        let version_manager =
-            rvm::VersionManager::new(false).map_err(|e| SolcError::Message(e.to_string()))?;
+        #[cfg(any(feature = "async", feature = "svm-solc"))]
```

Comment:
Maybe we should have similar logic to `solc`, if `async`, `svm-solc` not enabled then just local compilers only?